11ty/_includes/layouts/base.njk

```diff
@@ -7,10 +7,29 @@
     {# Use title with path, or append a space to the page title to avoid collpasing with the meta title #}
     {% set pageTitle = titleWithPath or title + ' ' or '' %}
     <title>{{ pageTitle + metadata.title }}</title>
+
+    <meta
+      name="twitter:description"
```

Comment:
twitter, of course, doesn't use the open graph standard because they're ~ special ~